python/tk_desktop/desktop_window.py

```diff
@@ -756,27 +785,22 @@ def __launch_app_proxy_for_project(self, project, pipeline_configuration_id=None
             else:
                 raise RuntimeError("Unknown platform: %s." % sys.platform)
 
-            if config_path is None:
-                engine.log_error("No path set for %s on the Pipeline "
-                                 "Configuration \"%s\" (id %d)." %
-                                 (current_platform,
-                                  pipeline_configuration["code"],
-                                  pipeline_configuration_id))
-
-                raise RuntimeError("The Toolkit configuration path has not\n"
-                                   "been set for your operating system.")
-
-            current_config_path = config_path
+            # FIXME: This logic should probably be built into core and be made public.
```

Comment:
totally agree that we should add this to core. Now seems like as good a time as ever!